src/ontology/cl-edit.owl

```diff
@@ -29057,15 +29057,16 @@ AnnotationAssertion(oboInOwl:id obo:CL_2000088 "CL:2000088")
 AnnotationAssertion(rdfs:label obo:CL_2000088 "Ammon's horn basket cell")
 EquivalentClasses(obo:CL_2000088 ObjectIntersectionOf(obo:CL_0000118 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001954)))
 
-# Class: obo:CL_2000089 (dentate gyrus of hippocampal formation granule cell)
+# Class: obo:CL_2000089 (dentate gyrus granule cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie") obo:IAO_0000115 obo:CL_2000089 "The principal cell type of the dentate gyrus.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie") Annotation(oboInOwl:hasDbXref "PMID:17765709") obo:IAO_0000115 obo:CL_2000089 "A granule cell that has soma location in the dentate gyrus cell layer of the hippocampal formation and has an elliptical cell body and characteristic cone-shaped tree of spiny apical dendrites. The branches extend throughout the molecular layer and the distal tips of the dendritic tree end just at the hippocampal fissure or at the ventricular surface. The dentate gyrus of hippocampal formation granule cell is the principal cell type of the dentate gyrus.")
```

Comment:
Consider using the new class label. Since the new class label is 'dentate gyrus granule cell', it can substitute 'dentate gyrus of hippocampal formation granule cell' in the definition as well.